packages/engine-formula/src/engine/dependency/dependency-tree.ts

```diff
@@ -63,18 +65,18 @@ export class FormulaDependencyTree extends Disposable {
 
     unitId: string = '';
 
-    rangeList: IUnitRangeWithToken[] = [];
+    rangeList: IUnitRange[] = [];
 
     formulaId: Nullable<string>;
 
     featureId: Nullable<string>;
 
-    isPassive: boolean = true;
+    isCache: boolean = false;
 
-    _childIds: Set<string> = new Set();
-
-    // the left top cell , which means the position of the FormulaDependencyTree
-    anchorToken: string = '';
+    constructor() {
+        super();
+        this.treeId = generateRandomId(8);
```

Comment:
maybe the id will meet a same one, we should separate two or more random string to make sure it is unique one.  Or like folowing code:

`
Golable.names = new Set();
let id = generateRandomId(8)
while(!Golable.names.has(id)){
   id =  generateRandomId(8)
}
return id;
`

Reply:
Done

Reply:
Done